src/ctx.c

```diff
@@ -432,3 +441,19 @@ void xmpp_ctx_free(xmpp_ctx_t * const ctx)
     xmpp_free(ctx, ctx); /* pull the hole in after us */
 }
 
+/** Set the timeout to use when calling xmpp_run().
+ *
+ *  @param ctx a Strophe context object
+ *  @param timeout the time to wait for events in milliseconds
+ *
+ *  @return the previously used timeout value
+ *
+ *  @ingroup Context
+ */
+unsigned long xmpp_ctx_set_timeout(xmpp_ctx_t * const ctx, const unsigned long timeout)
```

Comment:
Do we need to return previous timeout? I don't see usecase for this. Also indent should be 4 spaces according to coding style.